Update rds.md (#25443)
DataDog/documentation: content/en/database_monitoring/setup_postgres/rds.md

1. [Install and configure the Agent](#install-and-configure-the-agent)
1. [Install the RDS integration](#install-the-rds-integration)

> <div class="alert alert-info">Looking for a more streamlined setup where Datadog automatically deploys the agent to monitor your RDS? Click <a href="https://forms.gle/1dyK11ubXdmKuriH7">here</a> to share your interest.</div>

Reviewer: ```suggestion
<div class="alert alert-info">Interested in a more streamlined setup where Datadog automatically deploys the Agent to monitor your RDS? Click <a href="https://forms.gle/1dyK11ubXdmKuriH7">here</a> to share your interest.</div>
```
